examples/sentence-transformers-training/nli/README.md

```diff
@@ -46,6 +48,28 @@ For multi-card training you can use the script of [gaudi_spawn.py](https://githu
 HABANA_VISIBLE_MODULES="2,3" python ../../gaudi_spawn.py --use_deepspeed --world_size 2 training_nli.py bert-base-uncased
 ```
 
+
+# intfloat/e5-mistral-7b-instruct Model
+
+## Single-card Training with LoRA+gradient_checkpointing
+
+Due to the pretraining for intfloat/e5-mistral-7b-instruct needs around 130G memory so one hpu memory (98G) can not afford the training for this model. We can use LoRA + gradient_checkpointing to reduce the memory requirement for one hpu card. 
+
+```bash
+python training_nli_lora.py intfloat/e5-mistral-7b-instruct
+```
+
+## Multi-card Training with Deepspeed Zero2/3
+
+Due to the pretraining for intfloat/e5-mistral-7b-instruct needs around 130G memory so one hpu memory (98G) can not afford the training for this model. We need use the zero2/zero3 stage (model parallel) of deepspeed to reduce the memory requirement.
+
+We have tested that at least 4 hpu memories can satisfy the training of this model as below (Deepspeed Zero2).
+
+```bash
+python ../../gaudi_spawn.py --world_size 4 --use_deepspeed training_nli_deepspeed_zero2.py intfloat/e5-mistral-7b-instruct
+```
+In the command we need setup the lazy mode with the learning rate of 1e-7 and also deepspeed config of "ds_config.json". You need change the stage to 3 (Deepspeed Zero3) in "ds_config.json" if you want to reduce memory further. 
```

Comment:
```suggestion
In above command, we need to enable lazy mode with a learning rate of `1e-7` and configure DeepSpeed using the `ds_config.json` file. To further reduce memory usage, change the stage to 3 (DeepSpeed Zero3) in the `ds_config.json` file.
```

Reply:
@nngokhale since learning rate is important. Is it possible to point the user where they should change or create input arg for that? (personal suggestion)